src/libraries/Microsoft.PowerFx.Core/Functions/Delegation/DelegationStrategies/DelegationValidationStrategy.cs

```diff
@@ -223,7 +237,7 @@ public bool IsValidFirstNameNode(FirstNameNode node, TexlBinding binding, IOpDel
             Contracts.AssertValueOrNull(opDelStrategy);
 
             var isRowScoped = binding.IsRowScope(node);
-            var isValid = IsValidNode(node, binding);
+            var isValid = IsValidAsyncOrImpureNode(node, binding);
```

Comment:
>IsValidAsyncOrImpureNode [](http://example.com/codeflow?start=26&length=24)

IsValid would return true for Non-Async FirstNameNode before but that's no longer the case for IsValidAsyncOrImpureNode due to feature flag check AllowAsyncDelegation. You should fix that.

Reply:
yea, I already fixed it in my local build, but I've been testing against the unit tests in PowerApps-Client so I haven't pushed the latest changes